src/utils/data-fetcher-airtable.js

```diff
@@ -38,5 +39,45 @@ module.exports = {
 				}
 			});
 		});
+	},
+	populateResources: async () => {
+		return new Promise((resolve, reject) => {
+			const tagLabelToValue = {};
+			learnTags.resourceTags.forEach(tag => {
+				tagLabelToValue[tag.label] = tag.value;
+			});
+			const resources = [];
+			base("tbll2NpygbXrUnZZV").select().eachPage(function page(records, fetchNextPage) {
```

Comment:
@chosww could you add a code comment about how to find the base id with references to docs etc? Thanks.